superset-frontend/cypress-base/cypress/integration/dashboard/load.test.js

```diff
@@ -59,12 +64,19 @@ describe('Dashboard load', () => {
         requests.map(async xhr => {
           expect(xhr.status).to.eq(200);
           const responseBody = await readResponseBlob(xhr.response.body);
-          expect(responseBody).to.have.property('errors');
-          expect(responseBody.errors.length).to.eq(0);
-          const sliceId = responseBody.form_data.slice_id;
-          cy.get('[data-test="grid-content"]')
-            .find(`#chart-id-${sliceId}`)
-            .should('be.visible');
+          if (responseBody.result) {
+            responseBody.result.forEach(element => {
+              expect(element).to.have.property('error', null);
+              expect(element).to.have.property('status', 'success');
+            });
+          } else {
+            expect(responseBody).to.have.property('errors');
+            expect(responseBody.errors.length).to.eq(0);
+            const sliceId = responseBody.form_data.slice_id;
+            cy.get('[data-test="grid-content"]')
+              .find(`#chart-id-${sliceId}`)
+              .should('be.visible');
```

Comment:
I assume this assertion should be possible to do for V1 requests if we have `slice_id` readily available? If so, perhaps we should add it to the V1 request so the request params are the same for V1 and legacy requests.

Reply:
it is done